NSS-Bashful-Turtles/Yak-Social-Network
# Dp friends

Link to Ticket
a step in the right direction for  https://github.com/NSS-Bashful-Turtles/Yak-Social-Network/issues/11

Description of Proposed Changes
when a user is logged in it should display a list of the users friends. currently user 1 is hardcoded into the list but that can be changed later for other dependancies

Steps to Test
Outline the steps to test

git fetch --all
git checkout dpFriends
Run build

should show list of friends of user 1 in dom. 
this will conflict with the current index.js but the old changes can be elimitated as this is only for testing purposes

List general components of the application that this PR will affect:

Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
